### PR TITLE
Add simple script to assist in running tests

### PR DESCRIPTION
**Pull request by jugglinmike**

### CODE REVIEW

**Comment by rwaldron** [comment]
I was thinking about this last week and today. Here's what I'd actually like to see, for an "api":

``` sh
test262 "[runtime] --flags" TEST_PATTERN

# It's useful to test the V8 tests with flags to do reality checks on the assertion transformations:
test262 "v8 --harmony-arrays" array-

# Or bleeding edge "nodejs-like" runtimes
test262 "iojs --harmony-arrays" array-

# Also, would like to run tests against non-V8, eg. spidermonkey
test262 "js" array-
```

**Comment by jugglinmike** [comment]
@rwaldron I just pushed a commit that changes the script in the following ways:
1. Allows for direct specification of JavaScript runtimes
2. Runs only those tests that are "new" to the currently-checked-out branch of Test262

You asked for `#1`, but not for `#2`. Do you think that second part is a good idea?

**Comment by rwaldron** [comment]
I very much like #2 (also, sorry I didn't see this until now :|)

**Comment by rwaldron** [comment]
Can you add something to the readme that shows usages?

File: scripts/run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+RUNNER=tools/packaging/test262.py
+LOCAL_V8=v8-git-mirror/out/native/d8
+
+function help() {
+  cat <<HELP
+Usage: run-new-tests.sh [RUNTIME]
```
Comment (rwaldron) [inline scripts/run-tests.sh]: scripts/run-tests.sh?

Comment (jugglinmike) [inline scripts/run-tests.sh]: oops, thanks

**Comment by jugglinmike** [comment]
> Can you add something to the readme that shows usages?

Done!

**Pull request merged**